lightningd/peer_control.c

```diff
@@ -260,6 +260,12 @@ static struct bitcoin_tx *sign_last_tx(const tal_t *ctx,
 	return tx;
 }
 
+bool too_early_v2(const struct channel *channel)
+{
+	return channel->state == DUALOPEND_OPEN_INIT
+	       && !channel->last_tx;
+}
+
```

Comment:
No, this is exactly the kind of vague nomenclature I had to clean up.  If you add a new state, you don't need this.